### ISSUE TITLE
Add support for the sites that Moniturbate supports

### ISSUE THREAD

**Issue by Zero3K** [SagalVanderklok/ChatRoomRecorder#19]
It would be nice if you did that.

**Comment by SagalVanderklok** [comment]
What are the most popular sites after Chaturbate and BongaCams?

**Comment by Zero3K** [comment]
Cam4, Camsoda, Flirt4free, Streamate, Stripchat

**Comment by SagalVanderklok** [comment]
Yeah, adding support for additional websites is also planned, but as I have pretty tight schedule, it will require some time

**Comment by Zero3K** [comment]
Okay.

**Comment by Zero3K** [comment]
Any news regarding this?

**Comment by SagalVanderklok** [comment]
Stripchat support was added in 1.5.2 release

**Comment by Zero3K** [comment]
I saw. Thanks.

**Comment by Zero3K** [comment]
Why is this not planned?

**Comment by SagalVanderklok** [comment]
Oh, I've closed the issues because of inactivity. But actually I'm not sure that there're models streaming on Cam4/Camsoda/Flirt4free/Streamate and that aren't present on Chaturbate/StripChat/BongaCams

**Comment by Zero3K** [comment]
There are.

**Comment by SagalVanderklok** [comment]
Flirt4Free support was added in 1.6.4 release

**Comment by SagalVanderklok** [comment]
CamSoda support was added in 1.6.5 release

**Comment by maimaimottinhyeu22** [comment]
Can you please add Streamate? Thank you very much...

**Comment by maimaimottinhyeu22** [comment]
a great app

**Comment by maimaimottinhyeu22** [comment]
please add streamate.com thank you

**Comment by id7177681** [comment]
Could you add vr.stripchat.com VR version? Thank you.
